test/controllers/user_controller_test.exs

```diff
@@ -0,0 +1,29 @@
+defmodule ResearchResource.UserControllerTest do
+  use ResearchResource.ConnCase
+  alias ResearchResource.Repo
+  alias ResearchResource.User
+
+  test "GET /users/new", %{conn: conn} do
+    conn = get conn, "/users/new"
+    assert html_response(conn, 200) =~ "Create User"
+  end
+
+  test "POST /users/create - success", %{conn: conn} do
+    conn = post conn, user_path(conn, :create), user: %{
+      email: "me@test.com",
+      password: "secret",
+      first_name: "User",
+      last_name: "Test"
+    }
+    user = Repo.get_by!(User, %{email: "me@test.com"})
```

Comment:
I found it nicer to write just `Repo.get_by!(User, email: "me@test.com")`, you may find it nicer when you write more of these